src/Foundation/Site.php

```diff
@@ -25,7 +25,10 @@ public static function fromPaths(array $paths)
         date_default_timezone_set('UTC');
 
         if (! static::hasConfigFile($paths->base)) {
-            return new UninstalledSite($paths, $_SERVER['REQUEST_URI']);
+            return new UninstalledSite(
+                $paths,
+                Arr::get($_SERVER, 'REQUEST_URI', 'http://localhost')
```

Comment:
I like `bogus.url` a bit more as it's clearer that this doesn't actually mean anything. But either way, we'd want a comment explaining that this is only needed for validation at this stage, and doesn't actually do anything